src/test/java/ai/deepcode/javaclient/core/DeepCodeIgnoreInfoHolderTest.java

```diff
@@ -0,0 +1,278 @@
+package ai.deepcode.javaclient.core;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.function.Consumer;
+
+import static org.junit.Assert.*;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
Any reason to use ordering by test execution? For unit tests, execution order should be irrelevant.

Reply:
Yep, tests are independent here. But it's nice to see it executed in teh same order, so visually if some fail I can get what's wrong just by it place in the tests list. Just a personal preference and not affecting testing process.

Reply:
However, I would recommend to not use this annotation. Despite that it is not exist in JUnit5 (in case we'll upgrade later), the whole idea of unit testing is small, fast, independent, self-validating functions.

If the order of naming for helping purposes only (to find out where the issue happens in case of failure), maybe it's an indicator to provide a better name for test function or reduce the scope, make it smaller and self-explainable.

Reply:
fixed